app/controllers/comments_controller.rb

```diff
@@ -1,6 +1,13 @@
 class CommentsController < ApplicationController
   before_action :set_user_and_post, only: %i[new create]
 
+  def index_api
+    @comments = Comment.where(post_id: params[:post_id])
+    respond_to do |format|
+      format.json { render json: @comments }
+    end
+  end
+
   def new
     @comment = @post.comments.build
   end
```

Comment:
![image](https://github.com/stino-x/Blog-Rails-/assets/93039359/d06756c2-47be-46d0-bfba-cb8be46fb8f0)

- [ ] It would be nice if you address the issue I am facing while creating a comment using the API endpoint :thinking: Kindly fix it to make your app follow the project requirements :+1: .